### ISSUE TITLE
[Accessibility]: Dropdown is now being treating as menu, thus failing aria-required-children rule

### ISSUE THREAD

**Issue by yaananth** [OfficeDev/office-ui-fabric-react#3319]
[This ](https://github.com/OfficeDev/office-ui-fabric-react/commit/d57ca247d7333fa86473482e2037e1a5aca2c2f2) commit changed dropdown role from `combobox ` to `menu`.
But it fails https://aka.ms/keros/rules/axe.aria-required-children rule.
Because it has `menu` but no `menuitem` since we render the options with role `listbox` which has valid realtion ship with `combobox` but not a `menu`.

Commit says 

> Change combobox to menu as the dropdown is not editable but the screen reader keeps saying it is

I believe it can addressed with: http://www.w3.org/TR/wai-aria/states_and_properties#aria-autocomplete 

Can we revist and fix this please?

![image](https://user-images.githubusercontent.com/12280011/32461692-afefd354-c304-11e7-8e95-3220a7427bd9.png)

https://chrome.google.com/webstore/detail/keros-for-web-accessibili/pbjjkligggfmakdaogkfomddhfmpjeni

Run "fast pass"

**Comment by sugandhi-ms** [comment]
The change of role to "menu" from combobox also caused the narrator to announce the dropdown title and selected value twice on Microsoft Edge. 
A possible solution to the initial problem: " the dropdown is not editable but the screen reader keeps saying it is" can be adding aria-readonly=true.

**Comment by yaananth** [comment]
@rickyp-ms , @sugandhi-ms had a fix where aria-readonly is added,  does that fix your original intent of changing role?
Also I assume Dropdown component is never editable, I know Combox has editable as an optional props that can make it either, if that's the case, adding readonly for dropdown is probably fine.
